src/components/playback/playbackmanager.js

```diff
@@ -1409,14 +1409,23 @@ class PlaybackManager {
 
             const apiClient = ServerConnections.getApiClient(self.currentItem(player).ServerId);
 
-            apiClient.getEndpointInfo().then(function (endpointInfo) {
+            Promise.all([
+                apiClient.getEndpointInfo(),
+                apiClient.getCurrentUser()
+            ]).then(([endpointInfo, user]) => {
                 const playerData = getPlayerData(player);
                 const mediaType = playerData.streamInfo ? playerData.streamInfo.mediaType : null;
 
                 let promise;
                 if (options.enableAutomaticBitrateDetection) {
                     appSettings.enableAutomaticBitrateDetection(endpointInfo.IsInNetwork, mediaType, true);
                     promise = apiClient.detectBitrate(true);
+                } else if (user?.Policy
+                    && ((mediaType === 'Video' && !user.Policy.EnableVideoPlaybackTranscoding)
+                        || (mediaType === 'Audio' && !user.Policy.EnableAudioPlaybackTranscoding))) {
+                    // If transcoding is not allowed, we cannot use the specified bitrate - force bitrate detection
+                    console.warn('Transcoding is not allowed, cannot use the specified bitrate - force bitrate detection');
+                    promise = apiClient.detectBitrate(true);
```

Comment:
How does detecting the bitrate account for not being able to transcode? Does it just return `null`/`undefined` to indicate the source bitrate should be used?

Reply:
> How does detecting the bitrate account for not being able to transcode? Does it just return `null`/`undefined` to indicate the source bitrate should be used?

It just should be as in `Auto` mode - return detected bitrate or that 140Mbps (in the apiclient), i.e. greater than the original bitrate.
But if it is lower than the original bitrate it will probably fall back to remuxing. :thinking: 
I dunno if we can pass `undefined` as the maximum bitrate to the server. As an alternative, bitrate should be ignored on the server side.